### PR TITLE
feat(cloud.logs): iceberg support

### PR DESCRIPTION
**Pull request by pdepaepe**
Hello,

This PR will allow Iceberg support on Logs Data Platform API.

Thanks for your review, merge & prod time.

Signed-off-by: Pierre De Paepe <pierre.de-paepe@corp.ovh.com>

### CODE REVIEW

**Comment by antleblanc** [comment]
Thanks @pdepaepe for your contribution.

Do you have any requirements about using theses new services?

Because it seems that Iceberg isn't available for all regions and I'm not 100% sure that it safe to use in production.

@rbeuque74 could probably tell us a bit more about the current status I guess.

**Comment by pdepaepe** [comment]
> Do you have any requirements about using theses new services?
> 
> Because it seems that Iceberg isn't available for all regions and I'm not 100% sure that it safe to use in production.
> 

Iceberg is available in EU as /dbaas/logs today.

File: src/dbaas/logs/alert/alert.iceberg.service.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    var alertResource = iceberg("/dbaas/logs/:serviceName/output/graylog/stream/:streamId/alert/:alertId", {
+        serviceName: "@serviceName",
+        streamId: "@streamId",
+        alertId: "@alertId"
+    }, {
+        query: { method: "GET", isArray: true },
+        get: { method: "GET" },
+        post: { method: "POST" },
+        put: { method: "PUT" },
+        "delete": { method: "DELETE" }
```
Comment (marie-j) [inline src/dbaas/logs/alert/alert.iceberg.service.js]: If you don't override AngularJS `$resource` default configuration, those are not needed https://docs.angularjs.org/api/ngResource/service/$resource#$resource-returns

File: src/dbaas/logs/alias/alias.iceberg.service.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    // No cache here, because items can be shared at any moment by other users
+
+    var aliasResource = iceberg("/dbaas/logs/:serviceName/output/elasticsearch/alias/:aliasId", {
+        serviceName: "@serviceName"
+    }, {
+        create: { method: "POST" },
+        update: { method: "PUT" },
+        remove: { method: "DELETE" },
+        linkStream: { method: "POST", url: "/dbaas/logs/:serviceName/output/elasticsearch/alias/:aliasId/stream" },
+        unlinkStream: { method: "DELETE", url: "/dbaas/logs/:serviceName/output/elasticsearch/alias/:aliasId/stream/:streamId" },
```
Comment (marie-j) [inline src/dbaas/logs/alias/alias.iceberg.service.js]: Those should probably be in a `stream` submodule

File: src/dbaas/logs/alias/alias.iceberg.service.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        linkIndex: { method: "POST", url: "/dbaas/logs/:serviceName/output/elasticsearch/alias/:aliasId/index" },
+        unlinkIndex: { method: "DELETE", url: "/dbaas/logs/:serviceName/output/elasticsearch/alias/:aliasId/index/:indexId" },
```
Comment (marie-j) [inline src/dbaas/logs/alias/alias.iceberg.service.js]: Same here with `index`

File: src/dbaas/logs/alias/alias.iceberg.service.js (hunk 2 above)
Comment (pdepaepe) [reply src/dbaas/logs/alias/alias.iceberg.service.js]: Probably, but already the case on v6 https://github.com/ovh-ux/ovh-api-services/blob/master/src/dbaas/logs/alias/alias.v6.service.js

File: src/dbaas/logs/alias/alias.iceberg.service.js (hunk 3 above)
Comment (pdepaepe) [reply src/dbaas/logs/alias/alias.iceberg.service.js]: Same comment.

**Comment by pdepaepe** [comment]
Thank you @marie-j for your review, i did the modifications on usual methods.

**Pull request merged**